packages/uni_app/lib/session/logout/uni_logout_handler.dart

```diff
@@ -9,12 +10,21 @@ import 'package:url_launcher/url_launcher.dart';
 class UniLogoutHandler extends LogoutHandler {
   @override
   FutureOr<void> closeFederatedSession(FederatedSession session) async {
-    final logoutUri = session.credential.generateLogoutUrl();
+    // final appLinks = UniAppLinks();
+
+    // await appLinks.logout.intercept((redirectUri) async {
+    final logoutUri = session.credential.generateLogoutUrl(
+        // redirectUri: redirectUri,
+        );
+
```

Comment:
This code is left commented on purpose in case we can make the auto-close on logout work.